tutorial/westeros/westeros_addon_technologies.ipynb

```diff
@@ -0,0 +1,607 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, the newly added technology `po_turbine` needs to be classified as an `addon` technology"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scen.add_set('addon', 'po_turbine')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we need a new `type_addon`, which we will name `cogeneration_heat`.  We will classify the `po_turbine` via the *category* `addon` as one of the addon technologies as part of this specific `type_addon`.  In some cases, for example when modelling cooling technologies, multiple technologies can be classfied within a single `type_addon`.\n",
+    "\n",
+    "Via the set `map_tec_addon` we map the electricity generation technology, `coal_ppl`, to the `addon` technology, `po_turbine`. Multiple technologies, for example further fossil powerplants, could also be added to the this `type_addon` so as to be able to produce heat via the `po_turbine`.\n",
+    "\n",
+    "Note: the `addon` technology as well as the parent technology must have the same `mode`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type_addon = 'cogeneration_heat'\n",
+    "addon = 'po_turbine'\n",
+    "tec = 'coal_ppl'\n",
+    "scen.add_cat('addon', type_addon, addon)\n",
+    "scen.add_set('map_tec_addon', pd.DataFrame({'technology': tec,\n",
+    "                                            'type_addon': [type_addon]}))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last step required in order to link the `coal_ppl` is to define the `addon_conversion` factor between the `coal_ppl` and the `type_addon`.  This is important, because the `coal_ppl` generates electricty while the `po_turbine` generates heat. Therefore, we can use the inverse of the `input` coefficient from the `po_turbine`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Index for `addon_conversion` is ['node', 'technology', 'year_vtg',\n",
+    "#                                  'year_act', 'mode', 'time',\n",
+    "#                                  'type_addon', 'value', 'unit']\n",
+    "df = pd.DataFrame({'node': country,\n",
+    "                   'technology': tec,\n",
+    "                   'year_vtg': vintage_years,\n",
+    "                   'year_act': act_years,\n",
+    "                   'mode': 'standard',\n",
+    "                   'time': 'year',\n",
+    "                   'type_addon': type_addon,\n",
+    "                   'value': 5,\n",
```

Comment:
Please make sure this value 5 is explained. is this related to 0.2 input electricity of po_turbine?